src/pages/index.js

```diff
@@ -5,19 +5,112 @@ import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import PopupWithConfirm from "../components/PopupWithConfirm.js";
 import Section from "../components/Section.js";
 import UserInfo from "../components/UserInfo.js";
+import Api from "../components/Api.js";
 
 const editButton = document.querySelector('.profile__edit-button');
 const addButton = document.querySelector('.profile__add-button');
+const profileAvatar = document.querySelector('.profile__avatar-edit');
 
 const addNewCardForm = document.forms['card-form'];
 const profileEditForm = document.forms['profile-form'];
+const updateAvatarForm = document.forms['avatar-form'];
+
+const formInputName = document.querySelector('#profile-input-name');
+const formInputDescription = document.querySelector('#profile-input-description');
+
+// ! ||--------------------------------------------------------------------------------||
+// ! ||                                    User Info                                   ||
+// ! ||--------------------------------------------------------------------------------||
+const userInfo = new UserInfo({
+    nameEl: selectors.profileTitle, 
+    descriptionEl: selectors.profileDescription, 
+    avatarEl: selectors.profileAvatar,
+});
+
+// ! ||--------------------------------------------------------------------------------||
+// ! ||                                         API                                    ||
+// ! ||--------------------------------------------------------------------------------||
+const api = new Api({
+    baseUrl: "https://around-api.en.tripleten-services.com/v1",
+    headers: {
+      authorization: "d8e635dc-b9a3-40bb-8b8a-44b118a2b105",
+      "Content-Type": "application/json",
+    },
+});
 
-const profileNameInput = document.querySelector('#profile-input-name');
-const profileDescriptionInput = document.querySelector('#profile-input-description');
+let section;
+let currentUserId;
+
+Promise.all([api.fetchUserInfo(), api.getInitialCards()])
+    .then(([userData, cards]) => {
+        currentUserId = userData._id;
+        userInfo.setUserInfo({
+            name: userData.name,
+            description: userData.about,
+        });
+        userInfo.setAvatarImage(userData.avatar);
+        section = new Section(
+            {
+                items: cards,
+                renderer: (data) => {
+                    const card = renderCard(data);
+                    section.addItem(card);
+                }
+            },
+            selectors.cardsList
+        );
+        section.renderItems();
+    })
+    .catch((err) => {
+        console.log(err);
```

Comment:
`            console.log(err)
`

Good job that you `catch` possible errors at the end of server requests.


 You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors